pkg/core/xds/metadata.go

```diff
@@ -16,7 +16,7 @@ var metadataLog = core.Log.WithName("xds-server").WithName("metadata-tracker")
 const (
 	// Supported Envoy node metadata fields.
 
-	fieldDataplaneTokenPath         = "dataplaneTokenPath"
+	fieldDataplaneTokenPath         = "dataplane.token.filename"
```

Comment:
This change has to be backward compatible.

When CP uses this field, it should also support the old field

Reply:
You mean this will break the deployed `kuma-dp`'s that already are preconfigured with the old name?

Reply:
yes